docs/dml-of-carbondata.md

```diff
@@ -552,3 +553,50 @@ CarbonData DML statements are documented here,which includes:
   ```
   CLEAN FILES FOR TABLE carbon_table
   ```
+
+## CLEAN FILES
+
+  Clean files command is used to remove the Compacted and Marked
+  For Delete Segments from the store. Carbondata also supports Trash
+  Folder where all the stale data is moved to after clean files
+  is called
+
+  There are several types of compaction
+
+  ```
+  CLEAN FILES ON TABLE TableName
+  ```
```

Comment:
Clean files syntax needs to be changed from "clean files on table tablename" to  "clean files for table tablename" as testcases mentioned use  the above

Reply:
done